docs/formats.md

```diff
@@ -66,6 +66,7 @@ For compression formats, metadata cannot be preserved, as this information in mo
 | LZMA          | [compression/lzma.py][lzma-handler]         | [`7z`][lzma-extractor]          |
 | LZO           | [compression/lzo.py][lzo-handler]           | [`lzop`][lzo-extractor]         |
 | XZ            | [compression/xz.py][xz-handler]             | [`7z`][xz-extractor]            |
+| ZLIB          | [compression/zlib.py][zlib-handler]         | [`ZlibExtractor` custom code]zlib-extractor]        |
```

Comment:
There is a missing `[` so it's not rendered correctly.  
You can spot these kinds of errors by using `mkdocs serve` and looking at the generated documentation.

Reply:
good to know ! fixed.